manageiq-operator/README.md

```diff
@@ -55,53 +55,15 @@ $ oc create -f deploy/crds/manageiq_v1alpha1_manageiq_cr.yaml
 
 **ManageIQ Instance Example**
 
-> Deployments' resource requests here are tailered to make them fit into a crc cluster, change them according to your cluster's resource capacity*
+> The domain here will work for a Code Ready Containers cluster. Change it to one that will work for your environment.
```

Comment:
Please also change the CR name to `manageiq.org_v1alpha1_manageiq_cr.yaml` in L53